sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/ExtractBenchmark.scala

```diff
@@ -92,8 +92,9 @@ object ExtractBenchmark extends SqlBasedBenchmark {
     val intervalFields = Seq("YEAR", "MONTH", "DAY", "HOUR", "MINUTE", "SECOND")
     val settings = Map(
       "timestamp" -> datetimeFields,
-      "date" -> datetimeFields,
-      "interval" -> intervalFields)
+      "date" -> datetimeFields)
+      // TODO(SPARK-34938): Recover the benchmark of interval case
```

Comment:
cc @MaxGekk FYI

Reply:
Here is the fix https://github.com/apache/spark/pull/32035

Reply:
Thx